### ISSUE TITLE
Timecodes for "ML Zoomcamp 3.10 - Training Logistic Regression with Scikit-Learn"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#199]
Youtube video: https://www.youtube.com/watch?v=hae_jXe2fN0

**Comment by dimzachar** [comment]
0:00:00 - Training logistic regression with scikit learn.
0:02:00 - Explaining hard and soft predictions in ML.
0:03:56 - Soft predictions used to make churn decision.
0:05:59 - Selecting customers for promotional emails, validation accuracy
0:08:03 - Data frame shows churn prediction accuracy.
0:10:11 - Explaining boolean array and model accuracy.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
